vita3k/gui/src/settings_dialog.cpp

```diff
@@ -251,6 +259,12 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Select your preferred cpu backend.");
         ImGui::Spacing();
+        if (config_cpu_backend == CpuBackend::Dynarmic) {
```

Comment:
```suggestion
        if (config_cpu_backend == CPUBackend::Dynarmic) {
```
Sorry i review on my tablette lol i have forget is writed like that